### PR TITLE
Proper format paths

### PR DESCRIPTION
**Pull request by axilleas**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]
	

 

------------------------------

We welcome all PR’s though this doesn’t guarantee it will be accepted.

## Description:

Add backticks to paths so they are easier to discern.

## Benefits of this PR and context:

The mapping procedure is a hard enough topic, getting the paths in backticks will help visualizing the example better.

### CODE REVIEW

**Comment by freezurbern** [comment]
I believe you misspelled Remote Path Mappings.

**Comment by axilleas** [comment]
> I believe you misspelled Remote Path Mappings.

Oops, fixed! Thanks :rocket:

File: readme-vars.yml [hunk 1]
```diff
@@ -46,9 +46,9 @@ app_setup_block_enabled: true
...
   
-  Another item to keep in mind, is that within radarr itself, you should then map your torrent client folder to your radarr folder: Settings -> Download Client -> advanced -> remote path mappings. I input the host of my download client (matches the download client defined) remote path is /downloads/Movies (relative to the internal container path) and local path is /media/downloads/completed/Movies, assuming you have folders to seperate your downloaded data types.
+  Another item to keep in mind, is that within Radarr itself, you should map your torrent client folder to your radarr folder: **Settings -> Download Client -> Advanced Settings -> Remote Path Mappings**. The input the host of my download client (matches the download client defined) remote path is `/downloads/Movies` (relative to the internal container path), and local path is `/media/downloads/completed/Movies`, assuming you have folders to seperate your downloaded data types.
```
Comment (nemchik) [inline readme-vars.yml:51]: > The input the host of my download client

Not quite right.

Comment (axilleas) [reply readme-vars.yml]: Fixed, thanks.

**Comment by nemchik** [comment]
Thanks for this.

**Pull request merged**